src/coreComponents/dataRepository/ExecutableGroup.hpp

```diff
@@ -90,7 +90,7 @@ class ExecutableGroup : public dataRepository::Group
   virtual real64 GetTimestepRequest( real64 const time )
   {
     GEOSX_UNUSED_VAR( time );
-    return std::numeric_limits< integer >::max();
+    return 0.1 * std::numeric_limits< real64 >::max();
```

Comment:
I would say that 1e99 is sufficient

Reply:
Actually I would say that `1e99` is preferred.

Reply:
Sounds good, it's changed to 1e99 now

Reply:
Great. Thanks for the quick work.